libcloud/test/compute/test_ec2.py

```diff
@@ -429,23 +429,23 @@ def test_list_sizes(self):
                 self.assertTrue('m2.4xlarge' in ids)
 
             if region_name == 'us-east-1':
-                self.assertEqual(len(sizes), 61)
+                self.assertEqual(len(sizes), 67)
                 self.assertTrue('cg1.4xlarge' in ids)
                 self.assertTrue('cc2.8xlarge' in ids)
                 self.assertTrue('cr1.8xlarge' in ids)
                 self.assertTrue('x1.32xlarge' in ids)
             elif region_name == 'us-west-1':
-                self.assertEqual(len(sizes), 52)
+                self.assertEqual(len(sizes), 56)
```

Comment:
Cool thank you! I fixed it and a couple more and just pushed. Fyi I had to copy the secrets file to get it to import. I'm not sure if there are setup steps I'm supposed to follow but I just did the following:
`cp libcloud/test/secrets.py-dist libcloud/test/secrets.py`